src/modules/mavlink/mavlink_main.cpp

```diff
@@ -2510,6 +2510,48 @@ void Mavlink::check_radio_config()
 	}
 }
 
+void
+Mavlink::covariance_from_matrixurt_helper(float urt[21], matrix::SquareMatrix3f &matrix1,
+		matrix::SquareMatrix3f &matrix2)
+{
+	size_t index = 1, count = 1;
+
+	for (size_t i = 0; i < 6; i++) {
```

Comment:
Please comment this function such that the comments enable a reviewer to judge whether the code matches the intention.

Reply:
I will add an example yes